File: PYTHON PROJETOS/PROVA ANTIGA/PROVA 3/ATIV 03.py
#-----LER N-----#

def lerN():

    n = int(input("Digite um valor maior que 2: "))

    while n <= 2:

        print("Número inválido")
        n = int(input("Digite um valor maior que 2: "))
    
    return n 

#-----CALCULO-----#

def calc(n):

    somaTotal = 0 
    a = 2
    m = 3

    while a <= n:

        print("%d/%d" %(a,m))
        somaTotal = somaTotal + (a/m)
        a = a + 1
        m = m + 2

    return somaTotal

#-----PROGRAMA PRINCIPAL-----#

n = lerN()
somaTotal = calc(n)

print("total: %.2f" %somaTotal)

File: C/output/LISTA 5/ativ 03.c
#include <stdio.h>
#include <stdlib.h>
#include <windows.h> // acentuação
#include <math.h>
#include <time.h> 

//----------------LER NÚMERO------------------//

void lerNum(float *x, float *y){

    printf("Digite o valor de X: ");
    scanf("%f", x); 

    printf("Digite o valor de y: "); 
    scanf("%f", y); 
}

//----------------CALCULO------------------//

void conta(float x, float y){

    float cont, calc;
    cont = 0;
    calc = 0;  

    while (cont < x){

        calc = calc + y;
        cont = cont + 1; 
    }

    printf("Resultado: %.2f", calc);  
}

//----------------PROGRAMA PRINCIPAL------------------//

int main(){ 
    
    SetConsoleOutputCP(65001); 
    
    float  x, y; 

    lerNum(&x, &y); 
    conta(x, y); 

    return 0; 
}